contracts/payloads/IGP89/PayloadIGP89.sol

```diff
@@ -0,0 +1,530 @@
+                vault: LBTC_GHO_VAULT,
+                vaultType: VAULT_TYPE.TYPE_1,
+                supplyToken: lBTC_ADDRESS,
+                borrowToken: GHO_ADDRESS,
+                baseWithdrawalLimitInUSD: 10_000, // $10k
+                baseBorrowLimitInUSD: 10_000, // $10k
+                maxBorrowLimitInUSD: 15_000 // $15k
+            });
+
+            setVaultLimits(VAULT_LBTC_GHO); // TYPE_1 => 109
+
+            VAULT_FACTORY.setVaultAuth(LBTC_GHO_VAULT, TEAM_MULTISIG, true);
+        }
+    }
+
+    // @notice Action 2: Set dust limits for RLP-USDC DEX
+    function action2() internal isActionSkippable(2) {
+        address RLP_USDC_DEX = getDexAddress(28);
+
+        // RLP-USDC DEX
+        DexConfig memory DEX_RLP_USDC = DexConfig({
+            dex: RLP_USDC_DEX,
+            tokenA: RLP_ADDRESS,
+            tokenB: USDC_ADDRESS,
+            smartCollateral: true,
+            smartDebt: false,
+            baseWithdrawalLimitInUSD: 10_000, // $10k
+            baseBorrowLimitInUSD: 0, // $0
+            maxBorrowLimitInUSD: 0 // $0
+        });
+        setDexLimits(DEX_RLP_USDC); // Smart Collateral
+
+        DEX_FACTORY.setDexAuth(RLP_USDC_DEX, TEAM_MULTISIG, true);
+    }
+
+    // @notice Action 3: Set dust limits for wstUSR DEXes and vaults
+    function action3() internal isActionSkippable(3) {
+
+        {
+            address wstUSR_USDC_VAULT = getVaultAddress(110);
+
+            // [TYPE 1] wstUSR/USDC vault
+            VaultConfig memory VAULT_wstUSR_USDC = VaultConfig({
+                vault: wstUSR_USDC_VAULT,
+                vaultType: VAULT_TYPE.TYPE_1,
+                supplyToken: wstUSR_ADDRESS,
+                borrowToken: USDC_ADDRESS,
+                baseWithdrawalLimitInUSD: 10_000, // $10k
+                baseBorrowLimitInUSD: 10_000, // $10k
+                maxBorrowLimitInUSD: 15_000 // $15k
+            });
+
+            setVaultLimits(VAULT_wstUSR_USDC); // TYPE_1 => 110
+
+            VAULT_FACTORY.setVaultAuth(wstUSR_USDC_VAULT, TEAM_MULTISIG, true);
+        }
+
+        {
+            address wstUSR_USDT_VAULT = getVaultAddress(111);
+
+            // [TYPE 1] wstUSR/USDT vault
+            VaultConfig memory VAULT_wstUSR_USDT = VaultConfig({
+                vault: wstUSR_USDT_VAULT,
+                vaultType: VAULT_TYPE.TYPE_1,
+                supplyToken: wstUSR_ADDRESS,
+                borrowToken: USDT_ADDRESS,
+                baseWithdrawalLimitInUSD: 10_000, // $10k
+                baseBorrowLimitInUSD: 10_000, // $10k
+                maxBorrowLimitInUSD: 15_000 // $15k
+            });
+
+            setVaultLimits(VAULT_wstUSR_USDT); // TYPE_1 => 111
+
+            VAULT_FACTORY.setVaultAuth(wstUSR_USDT_VAULT, TEAM_MULTISIG, true);
+        }
+
+        {
+            address wstUSR_USDC_DEX = getDexAddress(27);
+
+            // wstUSR-USDC DEX
+            DexConfig memory DEX_wstUSR_USDC = DexConfig({
```

Comment:
this is not needed, WSTUSR USDC got replaced with WSTUSR USDT, same for T2 vault below